decidim-proposals/app/cells/decidim/proposals/proposals_picker_cell.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  module Proposals
+    # This cell renders the linked resource of a proposal.
```

Comment:
doesn't it render the proposals linkable to a given resource? instead of the linked resource (meeting, result...). Am I right?

Reply:
Yes, I copied from another file, thanks!